### PR TITLE
Fix transparent bg profile img bug [fixes 385]

### LINKED ISSUE #385: [MyMeta] Allow profile images with transparency
The current code for loading profile images puts a solid color behind the image:

This is useful for the autogenerated images:

Screenshot from 2021-03-05 14-26-55

But unnecessarily limits the creativity of users that have chosen an image.

The background behind the profile image of a user who has specified one via 3Box should be transparent.

### PR DESCRIPTION
**Pull request by chair28980**
Paired with @dysbulic @mquellhorst @luxumbra 

Fixes #385 

Transparent background on profile image no longer renders green.

Resolves https://github.com/MetaFam/TheGame/pull/409

### CODE REVIEW

**Comment by chair28980** [comment]
@alalonde @dysbulic 
Screen Shot 2021-06-25 at 7 10 31 AM

**Comment by chair28980** [comment]
@alalonde Looks like the app is building but the build is failing Vercel Michiel is mentioning Vercel as a culprit not playing nice with Hasura (or vice versa)?

**Comment by alalonde** [comment]
It appears that you just have to fix a linting issue for the tests to pass

**Comment by dysbulic** [comment]
It wants you to run `yarn lint --fix` to resolve the issue.

**Comment by chair28980** [comment]
@dysbulic @alalonde Looks like we're good to go! :)

**Comment by dysbulic** [comment]
The `Avatar` class is also used in: https://github.com/MetaFam/TheGame/blob/transparent-profile-2021-06-25-chair/packages/web/components/Player/Section/PlayerHero.tsx#L42-L47.

**Comment by chair28980** [comment]
@dysbulic I think it should be set now, let me know :)

File: packages/web/components/Player/Section/PlayerHero.tsx [hunk 1]
```diff
@@ -39,9 +40,8 @@ export const PlayerHero: React.FC<Props> = ({ player }) => {
   return (
     <ProfileSection>
       <VStack spacing={8}>
-        <Avatar
-          w={{ base: 32, md: 56 }}
-          h={{ base: 32, md: 56 }}
+        <PlayerAvatar
```
Comment (dysbulic) [inline packages/web/components/Player/Section/PlayerHero.tsx:43]: You should leave in the `h` and `w` attributes and replace `src` and `name` with `{...{ player }}`.

**Review by dysbulic — CHANGES_REQUESTED** [review]
You replaced the wrong attributes on `Avatar`.

File: packages/web/components/Player/Section/PlayerHero.tsx [hunk 2]
```diff
@@ -39,11 +39,10 @@ export const PlayerHero: React.FC<Props> = ({ player }) => {
   return (
     <ProfileSection>
       <VStack spacing={8}>
-        <Avatar
+        <PlayerAvatar
           w={{ base: 32, md: 56 }}
           h={{ base: 32, md: 56 }}
-          src={getPlayerImage(player)}
-          name={getPlayerName(player)}
+          {...{ player }}
```
Comment (dan13ram) [inline packages/web/components/Player/Section/PlayerHero.tsx:45]: you could just do
```suggestion
          player={player}
```
right?

**Review by dan13ram — APPROVED** [review]
minor feedback. otherwise lgtm :)

File: packages/web/components/Player/PlayerAvatar.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import { Avatar, AvatarProps } from '@metafam/ds';
+import React from 'react';
+
+import { PlayerFragmentFragment } from '../../graphql/autogen/types';
```
Comment (dan13ram) [inline packages/web/components/Player/PlayerAvatar.tsx:4]: these imports can also be simplified by using absolute paths.
```suggestion
import { PlayerFragmentFragment } from 'graphql/autogen/types';
```

File: packages/web/components/Player/PlayerAvatar.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+import {
+	getPlayerImage,
+	getPlayerName,
+	hasPlayerImage,
+} from '../../utils/playerHelpers';
```
Comment (dan13ram) [inline packages/web/components/Player/PlayerAvatar.tsx:9]:
```suggestion
} from 'utils/playerHelpers';
```

File: packages/web/components/Player/Section/PlayerHero.tsx (hunk 2 above)
Comment (chair28980) [reply packages/web/components/Player/Section/PlayerHero.tsx:45]: I think it works but, who wants to see player twice!? :) 

@dysbulic showed me this lil trick and i forgot about it.

**Pull request merged**